docs/migrating-from-asciidoc.adoc

```diff
@@ -0,0 +1,329 @@
+
+|Underlined titles
+|Underline length must match title length +/- 2 characters
+|Underline length must match title length +/- 1 character (Underlined titles are deprecated anyway)
+
+|+ifeval::[]+ 
+|Flexible
+|Compares the values of attributes only
+
+|Block delimiters
+|Delimiters do not have to match in length
+|The length of start and end delimiters for a block must match
+
+|Asciidoc cell formatting (e.g `a\|`)
+|`a` or `asciidoc`
+|`a` only
+
+|Deprecated tables
+|Yes
+|No (you don't want them anyway)
+
+|====
+
+
+[[migrate-deprecated]]
+=== Deleted and deprecated attributes
+
+// Based on the previous table, but a lot of things shown as not-implemented actually are.
+These are attributes that either no longer exist, work differently, or have better alternatives.
+
+[cols="<20,<20,<60"]
+|====
+|{ap} |{ad} |Notes
+
+
+|`big`, `small`, `underline`, `overline`, `line-through`, colors
+|_deprecated_
+|Character attributes to apply formatting directly. 
+Usually better to apply a role, then apply the formatting based on that role by using a stylesheet.
+
+// Checked
+|`infile`
+.2+|_not implemented_
+.2+|Provides the name and directory of the current document. 
+(Distinct from `docfile`, because `infile` may be an included document, and `docfile` is always the master document).
+No {ad} equivalent.
+
+// Checked
+|`indir`
+
+|`asciidoc`
+|`asciidoctor` 
+|{ap} sets `asciidoc` to show that it is the current processor. 
+{ad} sets `asciidoctor` instead.
+
+// Ref migration guide
+|`toc2`
+.2+|`toc`
+.2+|Combined in a single attribute, see <<toc>>.
+
+// Ref migration guide
+|`toc-placement`
+
+// Ref ap UG. I know it workls in ad tables; no ref to ad paragrphs so assume not implemented.
+|options="unbreakable"
+|
+|In {ad}, this only works for tables, not paragraphs.
+
+// No checked
+|options="pgwide"
+|_not implemented_
+|DocBook attribute to make tables full page width, whatever the current indent.
+No {ad} equivalent.
+
+// old table
+|`replacements2`
+|`post_replacements`
+|Renamed.
+
+// http://www.methods.co.nz/asciidoc/userguide.html#X39
+// an ugly hack so not implemented
+|`plaintext`
+|_not implemented_
+|{ap} uses this to suppress inline substitutions and retain block indents when importing large blocks of plain text.
+{ad} deliberately does not implement it; the closest {ad} equivalent is a source block.
+
+|specialwords
+|_not implemented_
+|In {ap}, applies special formatting to named text.
+In {ad} this could be implemented using an extension.
+
+// does this need an explanation?
+|presubs
+|-
+|Not required
+
+// old table says not, but...
+|halign, valign for table cells
+|Column and cell specifiers
+|See <<cell>>.
+
+// think this should be left in for completeness
+|sgml
+|_not implemented_
+|SGML is archaic and has been replaced by XML.
+
+|showcomments
+|_not implemented_
+|In {ap}, turns single line comments into DocBook "remark" elements.
+In {ad} use a pass block.
```

Comment:
This information got misplaced. What we should say about showcomments is that the Asciidoctor philosophy is that comments should not be passed on to the output. That's not what they are for.